bedrock/contentful/tests/test_contentful_api.py

```diff
@@ -466,3 +478,171 @@ def test__only_child():
     }
     assert _only_child(node, "text")
     assert not _only_child(node, "dummy-other")
+
+
+# These are what the rich_text_renderer library use for its own tests
+mock_simple_node = {"value": "foo"}
+mock_node = {"content": [{"value": "foo", "nodeType": "text"}]}
+mock_list_node = {"content": [{"content": [{"value": "foo", "nodeType": "text"}], "nodeType": "list-item"}]}
+mock_hyperlink_node = {
+    "data": {"uri": "https://example.com"},
+    "content": [{"value": "Example", "nodeType": "text", "marks": []}],
+}
+
+
+def test__StrongRenderer():
+    assert StrongRenderer().render(mock_simple_node) == "<strong>foo</strong>"
+
+
+def test__EmphasisRenderer():
+    assert EmphasisRenderer().render(mock_simple_node) == "<em>foo</em>"
+
+
+@patch("bedrock.contentful.api.get_current_request")
+def test__LinkRenderer__mozilla_link(mock_get_current_request):
+
+    mock_request = Mock()
+    mock_request.page_info = {"utm_campaign": "TEST"}
+    mock_get_current_request.return_value = mock_request
+    mozilla_mock_hyperlink_node = deepcopy(mock_hyperlink_node)
+    mozilla_mock_hyperlink_node["data"]["uri"] = "https://mozilla.org/test/page/"
```

Comment:
Just curious if the actual code uses `mozilla.org` or `www.mozilla.org`? Because the first redirect you'll get is if you go to the subdomain-less URL.

Reply:
The actual code has this
```
if "mozilla.org" in url.netloc and url.netloc != "www.mozilla.org":
```
and - you're right - it will 302 to www, so I'm _assuming_ this code is here for links from, say, blog.m.o. 

Will update the test with that assumption and add a note, for future selves